cmake/Modules/Findursa.cmake

```diff
@@ -0,0 +1,26 @@
+add_library(ursa UNKNOWN IMPORTED)
+
+set(URL "https://github.com/hyperledger/ursa/")
+set(VERSION "80bc3e772f7217c0a6c51507c0ddcd9c8bc77c0d")
+
+set(URSA_BUILD ${CMAKE_CURRENT_BINARY_DIR}/libursa)
```

Comment:
Other external libraries are installed in `${EP_PREFIX}`, so that library files are in `${EP_PREFIX}/lib` and includes are in `${EP_PREFIX}/include`. For example https://github.com/hyperledger/iroha/blob/8917810fd5ef8657a98da9db051f973c9ef73c39/cmake/Modules/Findpq.cmake#L44 I suggest to follow the same convention here.